_data/master/navbars/getting-started.yml

```diff
@@ -135,6 +165,16 @@ toc:
   section:
   - title: Overview
     path: /getting-started/openstack/
+  - title: Requirements
+    section:
+    - title: System 
+      path: /getting-started/openstack/reqs/system
+    - title: Public cloud
```

Comment:
I think there will be some differences. I need to think through it some more. Things like if / how we want to present content like using cloud-specific CNI plugins in Kubernetes, for example.

I _think_ we could simplify the "requriements" here so that there's a single "network requirements" or similar page that covers all three, so we don't need three separate docs. Then, we'd cover using cloud-specific features or CNI plugins as an "advanced" section that comes maybe in "Installation" or another section outside of "Getting started".

I don't have a real problem with duplicating them in the structure, because the content is all in a single file, it's just referenced from multiple places to make the user journey easier.